### ISSUE TITLE
ARIA Authoring practices content was moved

### ISSUE THREAD

**Issue by Thyumo** [mdn/content#28053]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

### What specific section or headline is this issue about?

Specifications

### What information was incorrect, unhelpful, or incomplete?

In the table with the links to the ARIA specification, the link to the Authoring Practices bring us a page which informs us that the content has been moved.

Thankfully, there are link to all the relevant content on the page itself but it would still be better to access the content directly. 

### What did you expect to see?

This [page](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/) should be shown directly

### Do you have anything more you want to share?

Here is a screen of the content moved page:

<img width="565" alt="Capture d’écran 2023-07-19 à 17 14 31" src="https://github.com/mdn/content/assets/34267920/be979919-d783-48a5-8bea-20132cf5e649">

This is actually the case for other MDN pages, like the one about the menu ARIA role. And for some (e.g. menu), there is no directly corresponding page, some other pages related to the role are suggested instead (e.g. menu -> menubar)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tooltip_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tooltip_role/index.md
* Last commit: https://github.com/mdn/content/commit/4228a8881f9ac08fd3a997445bdc8b8e6b8c05f9
* Document last modified: 2023-05-09T20:36:54.000Z

</details>

**Comment by sideshowbarker** [comment]
Yeah, unfortunately, it looks like the ARIA Authoring Practices Task Force decided to break all the https://w3c.github.io/aria-practices URLs rather than setting up redirects for them, as they should have done instead.

So we now have 44 https://w3c.github.io/aria-practices URLs in the MDN sources that must be manually replaced with https://www.w3.org/WAI/ARIA/apg/ equivalents.
